openadapt/record.py

```diff
@@ -466,12 +468,45 @@ def read_keyboard_events(
     terminate_event: multiprocessing.Event,
     recording_timestamp: float,
 ) -> None:
+
+      # create list of indices for sequence detection
+    stop_sequence_indices = []
+    for _ in config.STOP_SEQUENCES:
+        # one index for each stop sequence in config.STOP_SEQUENCES
+        stop_sequence_indices.append(0)
+        
     def on_press(event_q, key, injected):
         canonical_key = keyboard_listener.canonical(key)
         logger.debug(f"{key=} {injected=} {canonical_key=}")
         if not injected:
             handle_key(event_q, "press", key, canonical_key)
 
+        # stop sequence code
+        nonlocal stop_sequence_indices
+        global sequence_detected
+
+        for i in range(0, len(config.STOP_SEQUENCES)):
+            # check each stop sequence
+            stop_sequence = config.STOP_SEQUENCES[i]
+            # stop_sequence_indices[i] is the index for this stop sequence
+            # get canonical KeyCode of current letter in this sequence
+            canonical_sequence = keyboard_listener.canonical(
+                keyboard.KeyCode.from_char(
+                    stop_sequence[stop_sequence_indices[i]]))
```

Comment:
@angelala3252 Can you please run `black`?

Reply:
the default is 
```
            canonical_sequence = keyboard_listener.canonical(
                keyboard.KeyCode.from_char(stop_sequence[stop_sequence_indices[i]])
            )
```

Is this acceptable or would you prefer the format you wrote above?